elm.json

```diff
@@ -61,7 +61,7 @@
         "Nri.Ui.Pagination.V1",
         "Nri.Ui.Palette.V1",
         "Nri.Ui.Panel.V1",
-        "Nri.Ui.Pennant.V3",
+        "Nri.Ui.Pennant.V4",
```

Comment:
I think CI is failing because both these lines need to be here (so both are exposed and we can use them)

Reply:
Why, right. Fixed!